FAQ.md

```diff
@@ -26,4 +26,23 @@ Well nobody, and everyone. Pulsar is a volunteer and community driven project. W
 
 # How do I contact members of Pulsar?
 
-There are many ways, of course the classical Issues or Pull Requests on Repos is fine and well, additionally there are [discussions](https://github.com/orgs/pulsar-edit/discussions) org wide if needed. We are also all pretty active and responsive on our [Discord Server](https://discord.gg/7aEbB9dGRT), and messages there are always welcome. Lastly, we have a *new* [Subreddit](https://www.reddit.com/r/pulsaredit/) you can join as well.
+There are many ways, of course the classical Issues or Pull Requests on Repos is fine and well, additionally there are [discussions](https://github.com/orgs/pulsar-edit/discussions) org wide if needed. We are also all pretty active and responsive on our [Discord Server](https://discord.gg/7aEbB9dGRT), and messages there are always welcome. Additionally feel free to contact us through any of the below resources, or check out our [Org Wide Readme](https://github.com/pulsar-edit) as it's most up to date.
+
+* [GitHub Discussions](https://github.com/orgs/pulsar-edit/discussions)
+* [Discord Server](https://discord.gg/7aEbB9dGRT)
+* [Reddit](https://www.reddit.com/r/pulsaredit/)
+* [Mastodon](https://fosstodon.org/@pulsaredit)
+
+# Why don't you rewrite Pulsar in **insert Popular Web App Framework or Language Here**
+
+At this time, here is our current stance:
+
+Firstly, moving away from Node and Electron would mean a complete rewrite of the whole editor, something we do not have the manpower to do.
+
+Additionally moving away from these technologies will break every single package we have. The ecosystem of Atom and its community packages is a huge reason we wanted to keep it alive, rendering these completely useless removes one of the biggest advantages of the editor as a whole, and asking every single package author to rewrite their packages is not realistic.
+
+Our own packages would have to be rewritten along with all of our Native Packages (which are already some of the most cumbersome to maintain) which would no longer function in this new environment.
+
+Lastly, Atom is the editor that created Electron, while it may have it's idiosyncrasies, we feel that it seems almost wrong to ditch it for some hot new thing.
```

Comment:
```suggestion
Lastly, Atom is the editor that created Electron, while it may have its idiosyncrasies, we feel that it seems almost wrong to ditch it for some hot new thing.
```
sp.